src/utilities/processo-juridico/index.ts

```diff
@@ -21,3 +21,80 @@ export function format(processoJuridico: string) {
       return result;
     }, '');
 }
+
+export function isValid(processoJuridico: string): boolean {
+  if (processoJuridico.length !== LENGTH) {
+    return false;
+  }
+
+  const value = processoJuridico.split('');
+  const digit = value.splice(7, 2).join('');
+
+  let procJudNum = value.join('');
+  procJudNum = '00000000000000000' + procJudNum;
+  procJudNum = procJudNum.slice(procJudNum.length - 18);
+
+  let b1, b2, b3, b4, b5, b6, b7, b8, b9, b10, b11, b12, b13, b14, b15, b16, b17, b18;
```

Comment:
I think to make your logic easier to understand, you should split the process into functions. What do you think about it?

Reply:
Yeah, it makes sense. I don't understand too much about the mathematical process behind it. But I'll see what I can do.

Reply:
Can we translate `processoJuridico` to english? I know the lib is `brazilian` utils, but we keep source code in english.